src/FAQ.html

```diff
@@ -0,0 +1,30 @@
+<html>
+    <header>
+        FAQs
+    </header>
+    <body>
+        <h1>FAQs</h1>
+        <h2>How do I format the input data</h2>
+        <p>The input data should be an Excel file with 2 worksheets. The first worksheet holds the paired Day 0 – Day of Failure data, while the second worksheet holds additional background (unpaired) data. Additional background samples are not necessary for the algorithm to run, but the precision of the estimates will be improved the more background genotypes are included. For both datasets, rows are samples, and columns correspond to microsatellite loci. The columns are indexed by the number of alleles present for each locus. For example, the column “Polya_2” contains the 2nd allele for the “Polya” locus. The values of the cells contain the measured sample lengths for each sample. Paired samples should be labeled as SAMPLEID_D0 and SAMPLEID_DX, where SAMPLEID is the unique sample identification number, and X is the Day of Failure (eg. 28). The “Site” column should contain the name of the sentinel site where samples were collected; all samples from the same site are analyzed together by the algorithm. <br/><br/>
+        The “loci repeats” input parameter represents the vector of sizes of the tandem repeats for each marker. This is an inherent property of each microsatellite marker and is known a priori. For example, the PolyA microsatellite marker consists of repeated trimers of the form ATT, and its loci repeat value is therefore 3.<br/><br/>
```

Comment:
This file appears to be unused/replaced by the FAQ React Component; delete one of them and just use the other?